### ISSUE TITLE
[Bug] Switch component always animated even if a11y feature about animation disabling is used

### ISSUE THREAD

**Issue by pylapp** [Orange-OpenSource/ouds-ios#660]
# Description

As a user needing to slow animations of my app
I want the switch to not use animations when I toggle them 
So that the app using the switch component will be more usable for me

# Context

For buttons in loading state we disable the loading animations of the settings is defined in the app a11y settings.

We should do the same things for the switch: is user disabled animations, just move from on state to off state.

**Comment by cyndichin** [comment]
Sorry, feel free to unassign me, I am unable to tackle this at this time. Thank you!

**Comment by pylapp** [comment]
@codex Fix this issue.

Get the environment variable saying the animations have been disabled or not.

If disabled, remove animations for the switch components. The switch indicator must move in a blast to the opposite state, without animations.

**Comment by B3nz01d** [comment]
Issue finished during Sprint # 28
